### PR TITLE
GitHub Actions config

### PR DESCRIPTION
**Pull request by sdarwin**
GitHub Actions CI script, generated from the .travis.yml file. Please refer to https://github.com/CPPAlliance/githubactions for more information and instructions.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+
+jobs:
+  posix:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "TOOLSET=gcc COMPILER=g++-4.4 CXXSTD=98,0x Job 0"
+            buildtype: "boost"
+            packages: "g++-4.4"
```
Comment (urandon) [inline .github/workflows/ci.yml:22]: Is there any real reason to spent public CI resources on outdated g++4.4?
The same is related to other outdated and unsupported compilers

Comment (sdarwin) [reply .github/workflows/ci.yml:22]: The file was automatically generated based on the existing .travis.yml which seems to still have g++4.4.    You are welcome to delete any outdated compilers.

Comment (Lastique) [reply .github/workflows/ci.yml:22]: I don't think removing support for any compilers in Boost.Parameter was discussed or announced anywhere. I'd prefer as many compilers tested as possible.

**Comment by Lastique** [comment]
The support for GHA has already been added. Thanks.

**Pull request closed without merging**